pkgs/by-name/li/libblake3/package.nix

```diff
@@ -33,7 +37,10 @@ stdenv.mkDerivation (finalAttrs: {
       asl20
       cc0
     ];
-    maintainers = with lib.maintainers; [ fgaz ];
+    maintainers = with lib.maintainers; [
+      fgaz
+      silvanshade
```

Comment:
You shouldn't do this if you didn't add yourself to maintainer-list yet. You can either swap the two commits, or do this in the other commit

Reply:
Oops, yeah I missed that. Will fix.